src/ledger/LedgerTxnTrustLineSQL.cpp

```diff
@@ -56,35 +44,23 @@ std::shared_ptr<LedgerEntry const>
 LedgerTxnRoot::Impl::loadTrustLine(LedgerKey const& key) const
 {
     ZoneScoped;
-    std::string accountIDStr, issuerStr, assetStr;
-    getTrustLineStrings(key.trustLine().accountID, key.trustLine().asset,
-                        accountIDStr, issuerStr, assetStr,
-                        mHeader->ledgerVersion);
 
-    std::string extensionStr;
-    soci::indicator extensionInd;
-    std::string ledgerExtStr;
-    soci::indicator ledgerExtInd;
+    validateTrustLineKey(mHeader->ledgerVersion, key);
+
+    std::string accountIDStr = KeyUtils::toStrKey(key.trustLine().accountID);
+    auto asset = toOpaqueBase64(key.trustLine().asset);
 
+    std::string trustLineEntryStr;
     LedgerEntry le;
```

Comment:
Nitpick: (Same thing as in the `LiquidityPoolEntry` PR) can you move this closer to where it's used?

Reply:
Done